examples/micropython/Makefile

```diff
@@ -0,0 +1,23 @@
+# name of your application
+APPLICATION = micropython
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# select MicroPython package
+USEPKG += micropython
+
+# include boot.py as header
+BLOBS += boot.py
+
+# configure MicroPython's heap size
+CFLAGS += '-DMP_RIOT_HEAPSIZE=(8U*1024)'
+
+# MicroPython needs a larger stack
+CFLAGS += '-DTHREAD_STACKSIZE_MAIN=THREAD_STACKSIZE_DEFAULT*4'
```

Comment:
How do you determine this value ? Is it supposed to be configured by users ?
Same comment as above about moving this CFLAGS to the `Makefile.include` file of the package.

Reply:
> How do you determine this value ? Is it supposed to be configured by users ?

It depends, as any stacksize value. This one, I doubled it until stack size related crashes stopped.